bibliopixel/animation/sequence.py

```diff
@@ -1,14 +1,29 @@
 from . import collection
+import random as rand
 
 
 class Sequence(collection.Collection):
+    def __init__(self, layout, random=False, **kwds):
```

Comment:
The reason I just added it as a param is that I may want a regular sequence or I may want a random sequence. And I don't want auto_demo to have to chose a different classes depending on which I want. That'd be silly. There's only one option here, random or not. Simple enough.